console-subscriber/src/lib.rs

```diff
@@ -225,75 +353,164 @@ where
             || meta.target() == "tokio::task::waker"
         {
             self.waker_callsites.insert(meta);
+        } else if meta.name() == "runtime.resource" {
+            self.resource_callsites.insert(meta);
+        } else if meta.name() == "runtime.async_op" {
+            self.async_op_callsites.insert(meta);
+        } else if meta.target() == "tokio::resource::op" {
+            self.resource_op_callsites.insert(meta);
         }
```

Comment:
If we want to do that, we probably need to change the way `Callsites` work and drop the 32 bound.

Reply:
Are there likely more than 32 ops in Tokio? That doesn't sound crazy... Perhaps `Callsites` could get a const generic for the size of the array?

Reply:
#87 makes `Callsites` have a const generic parameter for length, so we can make bigger callsite arrays for resource ops.

if there are a _lot_ of resource ops, though, we may want to use a hashmap or something instead; there's probably a point at which linear search is no longer efficient...